CONTRIBUTING.md

```diff
@@ -162,20 +150,36 @@ cogito/
 └── test_repo_owner
 ```
 
-## Running the end-to-end tests
+## Running the integration tests
 
-We are finally ready to run also the end-to-end tests:
+We are finally ready to run also the integration tests:
 
-```console
-$ task test
+```
+$ task test-e2e
```

Comment:
Wouldn't it be more natural to call them `test-integration`? The doc is always talking of "integration tests" and suddenly the command to run them refers to end-to-end tests.

Reply:
done